### ISSUE TITLE
Filamentary Keratitis

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10907]
Preferred term label:
Filamentary Keratitis

Synonyms
Filamentary keratopathy, Filamentary corneal inflammation

Definition (free text, please give PubMed ID)
A condition characterized by the formation of filaments composed of degenerated epithelial cells and mucus attached to the corneal surface. It is associated with symptoms such as pain, foreign body sensation, photophobia, and tearing. Filamentary keratitis is often linked to underlying conditions such as dry eye disease, post-surgical ocular surface disorders, or systemic diseases like Sjögren syndrome.

Parent term (use hpo.jax.org/app)
Abnormal corneal epithelium morphology (HP:11495)

Diseases characterized by this term
Dry eye disease, Sjögren syndrome

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001184
- label: Filamentary keratitis
- parents: [Abnormal corneal epithelium morphology (HP:0011495)](https://hpo.jax.org/app/browse/term/HP:0011495)
- pmid: PMID:38857689
- def: A condition characterized by the formation of filaments composed of degenerated epithelial cells and mucus attached to the corneal surface. It is associated with symptoms such as pain, foreign body sensation, photophobia, and tearing. Filamentary keratitis is often linked to underlying conditions such as dry eye disease, post-surgical ocular surface disorders, or systemic diseases like Sjoegren syndrome.
- comment: 
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10907
- synonyms:
